src/modules/rewards/hooks/useDailyRewards.tsx

```diff
@@ -0,0 +1,130 @@
+// React and other libraries
+import { useCallback, useEffect, useState } from 'react';
+
+// hooks
+import { useGetRewardsActivities, useGetUserRewardsDetails, useSendRecentActivities } from 'queries';
+import { useAccount } from 'hooks';
+
+// helpers
+import { walletToCAIP10 } from 'helpers/w2w';
+import { checkTimeToCurrent, getActivityStatus, getDayNumber } from '../utils/getDailyActivityStatus';
+
+const useDailyRewards = () => {
+  const { account, isWalletConnected } = useAccount();
+
+  // State variables
+  const [activeItem, setActiveItem] = useState<any | null>(null);
```

Comment:
```suggestion
  const [activeItem, setActiveItem] = useState<Type | null>(null);
```

Please provide proper type